data-jdbc/src/test/java/io/micronaut/data/jdbc/postgres/PostgresEntityWithIdClassRepository.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2017-2020 original authors
```

Comment:
Not really important, but maybe we should just remove copyright from test classes? If not then update from 2020 to the current year.

Reply:
Probably, I just copied :/